.helm/ecamp3/templates/api_deployment.yaml

```diff
@@ -167,40 +140,36 @@ spec:
                   key: oauth-jubladb-base-url
             {{- end }}
           volumeMounts:
-            - mountPath: /var/run/php
-              name: php-socket
-            - mountPath: /srv/api/config/jwt/public.pem
+            - mountPath: /app/config/jwt/public.pem
               name: jwt-keypair
               subPath: public.pem
-            - mountPath: /srv/api/config/jwt/private.pem
+            - mountPath: /app/config/jwt/private.pem
               name: jwt-keypair
               subPath: private.pem
           lifecycle:
             preStop:
               exec:
                 command: ["/bin/sh", "-c", "/bin/sleep 1; kill -QUIT 1"]
           {{- end }}
-          {{- template "api.phpContainer" . }}
+          {{- template "api.container" . }}
           resources:
-            {{- toYaml .Values.php.resources | nindent 12 }}
+            {{- toYaml .Values.api.resources | nindent 12 }}
           readinessProbe:
-            exec:
-              command:
-                - docker-healthcheck
+            tcpSocket:
```

Comment:
Yeah, this is directly from the template.

Previously we had the following probes:
- php container: fcgi socket ping (in docker-healthcheck.sh)
- caddy container: tcp port 3001 (see further upwards in this file)

Now we only have the 2nd, because the 1st one doesn't exist anymore.

The HEALTHCHECK instruction in Dockerfile is ignored in Kubernetes and only used in local development environment. But why the probe here in Kubernetes was implemented differently is good question.

We could certainly also probe the /metrics endpoint, if this gives a better indication of the liveliness of the system.